cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1694,6 +1696,35 @@ def verify_rerun_course(self, source_course_key, destination_course_key, destina
         self.assertInCourseListing(source_course_key)
         self.assertInCourseListing(destination_course_key)
 
+    def test_rerun_course_val_success(self):
```

Comment:
I think it would be better to just build this into test_rerun_course_success. verify_rerun_course should assert the equality of the list of videos for the original and rerun.